web/src/components/RunDetailLayout/HeaderLeft.tsx

```diff
@@ -7,18 +7,17 @@ import * as S from './RunDetailLayout.styled';
 
 interface IProps {
   name: string;
-  testId: string;
   triggerType: string;
 }
 
-const HeaderLeft = ({name, testId, triggerType}: IProps) => {
+const HeaderLeft = ({name, triggerType}: IProps) => {
   const navigate = useNavigate();
   const {run} = useTestRun();
   const createdTimeAgo = Date.getTimeAgo(run?.createdAt ?? '');
 
   return (
     <S.Section $justifyContent="flex-start">
-      <S.BackIcon data-cy="test-header-back-button" onClick={() => navigate(`/test/${testId}`)} />
+      <S.BackIcon data-cy="test-header-back-button" onClick={() => navigate(-1)} />
```

Comment:
hmm I'm not sure we want this behavior. This is basically the back button functionality. Why do you think we should go back instead of going to the list page?

Reply:
Yeah I rolled it back, it was feeling weird haha